frontend/src/js/controllers/challengeHostTeamsCtrl.js

```diff
@@ -309,17 +309,44 @@
 
         vm.inviteOthers = function(ev, hostTeamId) {
             ev.stopPropagation();
-            // Appending dialog to document.body 
-            var confirm = $mdDialog.prompt()
-                .title('Invite others to this Team')
-                .textContent('Enter the email address of the person')
-                .placeholder('email')
-                .ariaLabel('')
-                .targetEvent(ev)
-                .ok('Send Invite')
-                .cancel('Cancel');
 
-            $mdDialog.show(confirm).then(function(result) {
+            $mdDialog.show({
```

Comment:
Please create a separate variable for this template like we did earlier with `confirm` variable